app/components/footnote_modal_component/footnote_modal_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<a href="#" data-toggle="modal" data-target="#<%= modal_id %>"><%= @title %>&nbsp;<span style="color: #007bff;"><%= icon %></span></a>
+<div>
+  <div class="modal fade" id="<%= modal_id %>" tabindex="-1" role="dialog" aria-labelledby="#<%= modal_id %>-title" aria-hidden="true">
+    <div class="modal-dialog modal-lg modal-dialog-centered" role="document">
+      <div class="modal-content">
+        <div class="modal-header" style="background: #232b49 !important; color: white;">
```

Comment:
All of the styling should be extracted out into an scss file that sits alongside the component.